librandom/exp_randomdev.h

```diff
@@ -40,7 +40,9 @@ namespace librandom
 Name: rdevdict::exponential - exponential random deviate generator
 Description: Generates exponentially distributed random numbers.
 
-  p(x) = lambda exp(-lambda*x), x >= 0.
+  p(x) =  lambda exp(-lambda*x), x >= 0 if lambda > 0
+  p(x) = -lambda exp(-lambda*x), x <= 0 if lambda < 0
```

Comment:
I would write this differently for clarity: Add a sentence above pointing out explicitly that negative lambda are permitted and then yield negative numbers, and then
```
For lambda < 0:
  p(x) = 0 for x >= 0
  p(x) = |lambda| exp ( -|lambda| |x| ) for x < 0
```
Since we are coercing the usual notation here, we should be pretty explicit. Ideally, we would want to draw a positive x and then multiply that with -1, but that will only become possible in the future (see #488 for ideas under consideration).